cg/services/run_devices/run_names/pacbio.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from cg.services.run_devices.run_names.service import RunDirectoryNamesService
+
+
+class PacbioRunDirectoryNamesService(RunDirectoryNamesService):
+
+    def get_run_names(self) -> list[str]:
+        """
+        Get all the run names from a run directory, assuming the run directory has only sequencing
+        folders with SMRTcells inside. The run names are in the form <sequencing_run>/<SMRTcell>:
+        Example:
+            r84202_20240913_121403/1_C01
+        """
```

Comment:
I think this docstring just adds confusion by trying to excuse us expecting a correct folder structure on Hasta. To me, the only relevant info is the structure of the run names, since we already know that we are getting the pacbio run names by the class and method name

Reply:
fixed